src/psyclone/tests/psyir/nodes/structure_reference_test.py

```diff
@@ -180,3 +180,20 @@ def test_reference_accesses():
         dref.reference_accesses(var_access_info)
     assert ("Dependency analysis has not yet been implemented for "
             "Structures." in str(err.value))
+
+
+def test_struc_ref_semantic_nav():
+    ''' Test the 'member' method of the StructureReference. '''
+    grid_type = symbols.StructureType.create([
+        ("nx", symbols.INTEGER_TYPE, symbols.Symbol.Visibility.PUBLIC)])
+    grid_type_symbol = symbols.TypeSymbol("grid_type", grid_type)
+    ssym = symbols.DataSymbol("grid", grid_type_symbol)
+    # Reference to scalar member of structure
+    sref = nodes.StructureReference.create(ssym, ["nx"])
+    assert sref.member is sref.children[0]
+    # Break the first child to check that we get the expected error
+    sref._children = ["broken"]
+    with pytest.raises(InternalError) as err:
+        sref.member
```

Comment:
You can prepend `_ = ` to avoid the pylint complain.

Reply:
Done.